x/tokenfactory/client/cli/tx.go

```diff
@@ -29,7 +29,7 @@ func GetTxCmd() *cobra.Command {
 func NewCreateDenomCmd() *cobra.Command {
 	return osmocli.BuildTxCli[*types.MsgCreateDenom](&osmocli.TxCliDesc{
 		Use:   "create-denom",
-		Short: "create a new denom from an account. (Costs osmo though!)",
```

Comment:
Maybe we should update this to say:

`(osmo to create tokens is charged through gas consumption)`

It still costs osmo to create tokens, but the mechanism has changed. wdyt?

Reply:
Thanks for feedback! I have updated docs as per your suggestion.